src/utils/efd.h

```diff
@@ -1,6 +1,7 @@
 /*
     Copyright (c) 2012-2013 Martin Sustrik  All rights reserved.
     Copyright 2015 Garrett D'Amore <garrett@damore.org>
+    Copyright (c) 2015-2016 Jack R. Dunaway.  All rights reserved.
```

Comment:
Neither am I.  But I *have* been advised by one.

On Mon, Apr 18, 2016 at 1:34 PM, Jack R. Dunaway <notifications@github.com>
wrote:

> In src/utils/efd.h
> <https://github.com/nanomsg/nanomsg/pull/636#discussion_r60129203>:
>
> > @@ -1,6 +1,7 @@
> >  /*
> >      Copyright (c) 2012-2013 Martin Sustrik  All rights reserved.
> >      Copyright 2015 Garrett D'Amore <garrett@damore.org>
> > +    Copyright (c) 2015-2016 Jack R. Dunaway.  All rights reserved.
>
> IANL :D
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nanomsg/nanomsg/pull/636/files/95b664f66e89dfbcc99877d3132a54b420c69cf5#r60129203>
>